docs/api/1.0.2.md

```diff
@@ -4,11 +4,11 @@
 
 ### cdc *<a target="_blank" href="https://wso2.github.io/siddhi/documentation/siddhi-4.0/#source">(Source)</a>*
```

Comment:
This is a generated class, do not commit

Reply:
Fixed with commit: dfdf273f8aa49429a4ff79b2ef25af926ff20e13